src/include/connection.h

```diff
@@ -268,6 +268,19 @@ struct __wt_connection_impl {
 	uint64_t  ckpt_time_recent;	/* Checkpoint time recent/total */
 	uint64_t  ckpt_time_total;
 
+	/* Checkpoint stats and verbosity timers */
+	struct timespec ckpt_start_time;/* Checkpoint start/stop time */
+	struct timespec ckpt_stop_time;
+					/* Checkpoint verbosity start time */
```

Comment:
We start this comment from the start of the line.

Reply:
This code comment is specific for the member variable, but as it won't fit in the same line of declaration I have put it in a line above. e.g. see https://github.com/wiredtiger/wiredtiger/blob/b284b2ff3768b05a30ee72eb14741947ea3fa09a/src/include/connection.h#L207

Reply:
@nehakhatri5 there are some examples of having parameter comments indented on separate lines, but we try to avoid doing it because it is ambiguous which line they are associated with. In this case I feel like a single block comment along with well named variables is enough.